src/skmysql.ml

```diff
@@ -1038,7 +1038,8 @@ module Table = struct
   let pp_column_name fmt (Column.Pack c) = Column.pp_spec_name fmt c
 
   let pp_column fmt column =
-    Fmt.pf fmt "@[%a %a%a@]" pp_column_name column pp_column_type column
+    (* Adding backticks to column name so columns can be named mysql key words *)
+    Fmt.pf fmt "@[`%a` %a%a@]" pp_column_name column pp_column_type column
```

Comment:
it scares me that this wasn't here before

Reply:
>just dont name a column "from" haha

but same